login-workflow/src/new-architecture/screens/ResetPasswordScreen/ResetPasswordScreen.tsx

```diff
@@ -52,19 +52,31 @@ export const ResetPasswordScreen: React.FC<ResetPasswordScreenProps> = (props) =
         }
     }, [setIsLoading, setShowSuccessScreen, actions, code, passwordInput, email]);
 
-    const {
-        PasswordProps: passwordProps = {
-            newPasswordLabel: t('bluiAuth:CHANGE_PASSWORD.NEW_PASSWORD'),
-            confirmPasswordLabel: t('bluiAuth:CHANGE_PASSWORD.CONFIRM_NEW_PASSWORD'),
-            passwordNotMatchError: t('bluiCommon:FORMS.PASS_MATCH_ERROR'),
-            passwordRequirements: passwordRequirements,
-            passwordRef,
-            confirmRef,
+    const areValidMatchingPasswords = useCallback((): boolean => {
```

Comment:
Because of this method, we are always checking hardcoded password requirements. 
if we pass like `passwordRequirements:[ { regex: /^.{5,10}$/, description: '5-10 Characters', }, ],` it will match password and confirm password but next button will not enable due to `areValidMatchingPasswords()`.
<img width="453" alt="Screenshot 2023-07-24 at 11 50 00 AM" src="https://github.com/etn-ccis/blui-react-workflows/assets/133877691/7213eace-9ab8-48e5-8529-6e2866a7d286">

Reply:
Can you check in ResetPassword screen ?